shiny/ui/_input_update.py

```diff
@@ -475,21 +474,36 @@ def update_date(
 
     Note
     ----
-    {note}
+    A special value of `""` (an empty string) can be used to clear the value, min, and
```

Comment:
Should keep this note and add an additional note (below it)

```suggestion

    Note
    -----
    A special value of `""` (an empty string) can be used to clear the value, min, and
```